build.proj

```diff
@@ -41,11 +24,9 @@
     </Project>
     <!- signing must happen before packaging -!>
     <Project Include="src\sign.builds" />
-    -->
     <Project Include="src\packages.builds" Condition="'$(BuildPackages)'=='true'">
```

Comment:
My expectation is that we will only build packages via the build-packages.cmd command and not part of build.cmd. At least that is how I've spec'ed it to this point. Do you see any issues with doing that as an independent step?